Marlin/Marlin.h

```diff
@@ -365,4 +372,13 @@ extern uint8_t active_extruder;
 
 void calculate_volumetric_multipliers();
 
+// Buzzer
+#if HAS_BUZZER
+  #if ENABLED(SPEAKER)
+    extern Speaker buzzer;
```

Comment:
When I made the singleton classes, I actually included the `extern` line in the class headers. If you feel like doing that instead, it's a smidgen more encapsulated.